aclgen.py

```diff
@@ -181,14 +189,23 @@ def RenderFile(input_file, output_directory, definitions,
     logging.warn('bad file: \n%s', e)
     raise
 
+  filename, file_extension = os.path.splitext(input_file)
+  format_dict = {
+    '.yml': yamlpolicyparser,
+    '.pol': policyparser
+  }
+  if file_extension not in format_dict:
+    raise Error('bad policy file extension ' + file_extension)
```

Comment:
Please create a specific Error class and add this to the docstring for RenderFile